### ISSUE TITLE
Learn how to compile all .js files into one dotsynth.js

### ISSUE THREAD

**Issue by Awk34** [Awk34/dotsynth#24]
look it up mane

**Comment by uber5001** [comment]
How is this a performance issue? Compiling all the scripts into one script will make downloading new versions **slower**, since users have to re-download everything.

The `defer` attribute (on script tags) already makes all the scripts load in parallel, and executes them in order. The only change that I could see being made here is adding the `async` attribute, which is the same thing, but no guaranteed order.

So... why do we need this?

**Comment by Awk34** [comment]
When have you ever seen an app distributed in 10 JS files? I guess we are probably never going to distribute this, but still. We are not going to push changes to the main app site very often. A minified, singular "dotsynth.js" would be better in my opinion. The separate files are better for development. If you disagree, feel free to mark this issue with "wontfix".

**Comment by uber5001** [comment]
Herp derp. We need to do this when we minify it for major releases...

Yeah. We'll probably do this for .synth 1.0.
